src/Form/Field/ContentTypeFieldChoiceList.php

```diff
@@ -52,39 +63,52 @@ public function getStructuredValues()
     }
 
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
```

Comment:
remove {@inheritDoc} ?

Reply:
Referering to: Symfony\Component\Form\ChoiceList\ChoiceListInterface methods